gnome-extra/gnome-commander/gnome-commander-1.14.0.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+GNOME2_LA_PUNT="yes"
+
+inherit gnome2 optfeature
+
+DESCRIPTION="A graphical, full featured, twin-panel file manager"
+HOMEPAGE="https://gcmd.github.io/"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="chm exif gsf pdf samba taglib test +unique"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	app-text/yelp-tools
+	dev-libs/atk
```

Comment:
I see no evidence that this dependency is correct.

Reply:
there was this commit c32dfd87cbb from @juippis where this dependency was added. I did not find the time to check why this was done, maybe he can comment. But I agree that it can be removed.

Reply:
grep -i atk /var/db/pkg/gnome-extra/gnome-commander-1.14.0/NEEDED.ELF.2
X86_64;/usr/lib64/gnome-commander/plugins/libfileroller.so;libfileroller.so;;libgtk-x11-2.0.so.0,libgdk-x11-2.0.so.0,libpangocairo-1.0.so.0,**libatk-1.0.so.0**,libcairo.so.2,libgdk_pixbuf-2.0.so.0,libgio-2.0.so.0,libpangoft2-1.0.so.0,libpango-1.0.so.0,libgobject-2.0.so.0,libglib-2.0.so.0,libharfbuzz.so.0,libfontconfig.so.1,libfreetype.so.6,libstdc++.so.6,libc.so.6,libgcc_s.so.1;x86_64

scanelf -n /usr/lib64/gnome-commander/plugins/libfileroller.so
 TYPE   NEEDED FILE
ET_DYN libgtk-x11-2.0.so.0,libgdk-x11-2.0.so.0,libpangocairo-1.0.so.0,**libatk-1.0.so.0**,libcairo.so.2,libgdk_pixbuf-2.0.so.0,libgio-2.0.so.0,libpangoft2-1.0.so.0,libpango-1.0.so.0,libgobject-2.0.so.0,libglib-2.0.so.0,libharfbuzz.so.0,libfontconfig.so.1,libfreetype.so.6,libstdc++.so.6,libc.so.6,libgcc_s.so.1 /usr/lib64/gnome-commander/plugins/libfileroller.so

Reply:
Hi! It took a while but since yesterday, [gnome-commander-1.16.0](https://github.com/gentoo/gentoo/blob/master/gnome-extra/gnome-commander/gnome-commander-1.16.0.ebuild) is in portage which does not have this issue with that many linked libraries anymore - as the program uses Meson now. Example:
```
$ scanelf -n /usr/lib64/gnome-commander/plugins/libfilerollerplugin.so
 TYPE   NEEDED FILE 
ET_DYN libglib-2.0.so.0,libgio-2.0.so.0,libgobject-2.0.so.0,libgtk-x11-2.0.so.0,libgdk_pixbuf-2.0.so.0,libstdc++.so.6,libgcc_s.so.1,libc.so.6 /usr/lib64/gnome-commander/plugins/libfilerollerplugin.so
```

I'm happy :smiley: